Project.toml

```diff
@@ -5,11 +5,14 @@ version = "5.5.0"
 
 [deps]
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+CellListMap = "69e1c6dd-3888-40e6-b3c8-31ac5f578864"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
+InteractiveDynamics = "ec714cd0-5f51-11eb-0b6e-452e7367ff84"
```

Comment:
WOOPS, I just saw this. All of these MUST be removed. You want to add CellListMap in the `docs/Project.toml`. NOT the top most level project toml, these are the formal package dependencies!!!!

Reply:
Yes, I know, it was not the idea to merge this now. I have yet to finish and document the new interface and release the CellListMap side. 

(The problem here is that dev'ing two packages at once is cumbersome without adding the dependency one).